### PR TITLE
Small fixes to automated branch creation on Issue assignment

### PR DESCRIPTION
**Pull request by ClaudiaComito**
## Due Diligence

- General:
    - [x]  **base branch** must be `main` for new features, latest release branch (e.g. `release/1.3.x`) for bug fixes
    - [x]  **title** of the PR is suitable to appear in the [Release Notes](https://github.com/helmholtz-analytics/heat/releases/latest)
- Implementation:
    - [x] unit tests: all split configurations tested
    - [x] unit tests: multiple dtypes tested
    - [x] documentation updated where needed

## Description

Small fixes to automated branch creation on Issue assignment. 

Issue/s resolved: #1182 

## Changes proposed:
- introduced `workflows` prefix
- modified branch naming scheme to match what we used so far
- updated Quick_start.md accordingly
-

## Type of change

Documentation update
## Memory requirements

NA
## Performance

NA

#### Does this change modify the behaviour of other functions? If so, which?
 no

### CODE REVIEW

File: quick_start.md [hunk 1]
```diff
@@ -67,31 +67,26 @@ Local torch tensor on rank  1 :  tensor([5, 6, 7, 8, 9], dtype=torch.int32)
 
 ## New Contributors
 
-1. [Fork](https://docs.github.com/en/get-started/quickstart/contributing-to-projects) or, if you have write access, clone the [Heat repository](https://github.com/helmholtz-analytics/heat).
+1. Pick an Issue you'd like to work on. Check out [Good First Issues](https://github.com/helmholtz-analytics/heat/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22), start from most recent. Feel free to assign to yourself.
```
Comment (mrfh92) [inline quick_start.md:70]: Can externals assign themselves? When I wanted to assign Saj to the FFT, he had to comment first in order to appear in the list of people I could assign... it seems that he could not assign himself and that I couldnt assign externals who did not already appear in the issue (e.g. by commenting etc.)

**Review by mrfh92 — DISMISSED** [review]
Looks fine to me; I only have one comment (see above).

File: quick_start.md (hunk 1 above)
Comment (ClaudiaComito) [reply quick_start.md:70]:
```suggestion
1. Pick an Issue you'd like to work on. Check out [Good First Issues](https://github.com/helmholtz-analytics/heat/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22), start from most recent. Get in touch and ask to be assigned to the issue.
```

Comment (ClaudiaComito) [reply quick_start.md]: Good catch, addressed, thanks Fabian!

**Review by mrfh92 — APPROVED** [review]
:+1:

**Pull request merged**